rocAL/rocAL/source/api/rocal_api_data_loaders.cpp

```diff
@@ -2534,3 +2552,143 @@ rocalResetLoaders(RocalContext p_context)
     }
     return ROCAL_OK;
 }
+
+RocalTensor  ROCAL_API_CALL
+rocalAudioFileSourceSingleShard(
+        RocalContext p_context,
+        const char* source_path,
+        const char* source_file_list_path,
```

Comment:
To remove this file_list_path now - will be added as a part of the file list reader (meta-data reader)